sopel/modules/meetbot.py

```diff
@@ -221,7 +224,7 @@ def meetingsubject(bot, trigger):
     logfile.write('</ul><h3>' + trigger.group(2) + '</h3><ul>')
     logfile.close()
     logplain('Current subject: ' + trigger.group(2) + ', (set by ' + trigger.nick + ')', trigger.sender)
-    bot.say('Current subject: ' + trigger.group(2))
+    bot.say(bold('Current subject: ') + trigger.group(2))
```

Comment:
GitHub's not showing me the control characters, but checking local source code tells me that the original code bolded only `Current subject`, without the trailing text.

It's probably better to include the `:`, but if I was going to nitpick (and you know me, I will) I'd ask to exclude the space from formatting because it doesn't make sense to include there.